exercises/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,9 @@
 */
 
```

Comment:
What could have been added here that would have explained the purpose of the code below and referred back to the exercise? Comments :D. Always try to add a comment to explain code. A good comment could be :
//Lines of code below demonstrate that Javascript statements can have a boolean value of true or false.